test/conformance/kuttl/generate/clusterpolicy/cornercases/trigger-resource-name-exceeds-63-characters/01-policy.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name:  generate-network-policy
+spec:
+  rules:
+  - name: default-deny
+    match:
+      any:
+      - resources:
+          kinds:
+          - ConfigMap
+    generate:
+      apiVersion: networking.k8s.io/v1
+      kind: NetworkPolicy
+      name: default-deny
+      namespace: "{{request.object.metadata.namespace}}"
+      synchronize: false
```

Comment:
Change to `true` and also a step which deletes the trigger to ensure the downstream is also deleted.

Reply:
Done.